artiq/gateware/targets/sayma_rtm.py

```diff
@@ -110,6 +113,8 @@ def __init__(self, platform):
         csr_devices = []
 
         self.submodules.crg = CRG(platform)
+        csr_devices.append("crg")
```

Comment:
Why not use the GPIO out core instead of adding this to the CRG? CRG is normally used to clock the local design only.

Reply:
Lack of familiarity with misoc/to the man with a hammer everything is a nail.

Reply:
Do you mind changing this when merging?